pypeit/images/combineimage.py

```diff
@@ -289,8 +290,10 @@ def run(self, bias=None, flatimages=None, ignore_saturation=False, sigma_clip=Tr
             comb_basev = np.ma.sum(np.ma.MaskedArray(basev_stack, mask=bpm_stack),axis=0).filled(0.)
             # Convert to standard error in the median (pi/2 factor relates standard variance
             # in mean (sum(variance_i)/n^2) to standard variance in median)
-            comb_rn2[gpm] *= np.pi/2/nstack[gpm]**2/comb_scl[gpm]**2
-            comb_basev[gpm] *= np.pi/2/nstack[gpm]**2/comb_scl[gpm]**2
+            comb_rn2[gpm] *= np.pi/2/nstack[gpm]**2
+            comb_basev[gpm] *= np.pi/2/nstack[gpm]**2
+            # Divide by the number of images that contributed to each pixel
+            comb_scl[gpm] *= np.pi/2/nstack[gpm]
```

Comment:
Do we need the factor of pi/2 here?

Reply:
Thanks @kbwestfall for reviewing this. 
Correct me if I'm wrong, but as it says in the comment in lines 291-292, pi/2 "_relates the standard variance in mean (sum(variance_i)/n^2) to standard variance in median_". So according to this:
`V_m = pi/2 * sum(V_i)/N^2 =  pi/2 * sum(s_i f_i)/N^2 + pi/2 * sum(V_base_i)/N^2 + pi/2 epsilon^2*sum(f_i^2)/N^2`

The second term (` pi/2 * sum(V_base_i)/N^2`) is done in line 294
Adding the pi/2 factor in `comb_scl` should take care of the first term.
But now I'm thinking that also the third term (the one with the noise floor) should have it. Oh boy, that would be more complicated to add.
What do you think?

Reply:
I guess for the pi/2 factor in the third term, we can ignore it as we do with N

Reply:
Yeah, that all makes sense to me.  Let's ignore it in the third term, and keep the pi/2 factor in the first term.